docs/transmission.md

```diff
@@ -0,0 +1,89 @@
+# Transmission model
+
+## Overview
+
+The transmission model controls person-to-person transmission between an
+infectious agent and other susceptible agents. Agents become infectious immediately
+upon successful infection attempt, though they may have a transmission probability of
+0 for some time (mimicking an exposed compartment). Once infectious, agents have a pre-assigned
+number of infection attempts. Infection attempt times are drawn from the generation interval.
+Once there are no more infection attempts remaining for the agent, we argue that they do not
+transition to a recovered compartment but rather back to a susceptible compartment but
+with high immunity.
+
+## Workflow
+
+The transmission model is centered around a key workflow:
+1. Watch for an agent becoming infectious (S --> I transition).
+2. For the infectious agent, draw a number of infection attempts. This can be from the
```

Comment:
What does this mean? Ri vs contact distribution?

Reply:
Number of infection attempts could be a draw from a distribution over $R_i$ or the number of infection attempts could be some function of the contact network of the transmitter.